simulator/examples/triangle-contains.rs

```diff
@@ -0,0 +1,130 @@
+//! # Example: Triangles and `contains()`
```

Comment:
I would keep the example, but change it to be more generic and demonstrate `contains` for all primitives that support it. But this doesn't need to be in this PR.

Reply:
I've elected to remove this example, at least for now. I think creating an example that demonstrates the `contains()` behaviour specifically, and not a triangle debugger, is a better way to go. That can happen in a new PR.